core/runtime/src/main/java/org/eclipse/sw360/antenna/frontend/MetaDataStoringProject.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.frontend;
+
+import org.eclipse.sw360.antenna.api.IProject;
+
+import java.io.File;
+
+public abstract class MetaDataStoringProject implements IProject {
+    protected File propertiesFile;
+    protected Build build;
+
+    public abstract void setProjectId(String id);
```

Comment:
Is this used anywhere?

Reply:
Is used here https://github.com/eclipse/antenna/blob/master/core/frontend-stubs/cli-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/cli/AntennaCLISettingsReader.java#L161